src/components/common/ListShareClass.vue

```diff
@@ -236,7 +244,7 @@ export default class ListShareClass extends Vue {
    * @param direction The direction of the move
    * @param seriesIndex The index of the series
    */
-  protected moveIndex (indexFrom: number, direction: string, seriesIndex = -1): void {
```

Comment:
This change and the ones below is to fix the method visibility warnings in the template.